api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/DosageTranslatorImpl.java

```diff
@@ -51,6 +52,12 @@ public Dosage toFhirResource(@Nonnull DrugOrder drugOrder) {
 		dosage.setRoute(conceptTranslator.toFhirResource(drugOrder.getRoute()));
 		dosage.setTiming(timingTranslator.toFhirResource(drugOrder));
 		
+		if (drugOrder.getInstructions() != null) {
+			CodeableConcept additionalInstructions = new CodeableConcept();
+			additionalInstructions.setText(drugOrder.getInstructions());
+			dosage.addAdditionalInstruction(additionalInstructions);
```

Comment:
This could be written:

```suggestion
			dosage.addAdditionalInstruction().setText(drugOrder.getInstructions());
```

But it's functionally equivalent.

Reply:
Strange, I thought that wasn't working for some reason, but it looks like it does... :)